unstructured/ingest/connector/fsspec.py

```diff
@@ -111,13 +124,77 @@ def get_file(self):
             **self.config.get_access_kwargs(),
         )
         logger.debug(f"Fetching {self} - PID: {os.getpid()}")
-        fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+        try:
+            fs.get(rpath=self.remote_file_path, lpath=self._tmp_download_file().as_posix())
+            self.file_exists = True
+        except Exception as e:
+            if isinstance(e, FileNotFoundError):
+                self.file_exists = False
+            raise
```

Comment:
I don't think we really need to be updating file_exists in get_file. by the time the process is calling get_file the parent process has already already checked if file_exists